spec/app/domain/authentication/authn_k8s/inject_client_cert_spec.rb

```diff
@@ -0,0 +1,294 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Authentication::AuthnK8s::InjectClientCert do
+  let(:spiffe_name) { "SpiffeName" }
+  let(:spiffe_namespace) { "SpiffeNamespace" }
+
+  let(:account) { "SomeAccount" }
+  let(:service_id) { "ServiceName" }
+  let(:common_name) { "CommonName.#{spiffe_namespace}.Controller.Id" }
+  let(:csr) { "CSR" }
+
+  let(:host_id) { "HostId" }
+  let(:host_id) { "HostId" }
+  let(:host_role) { double("HostRole", :id => host_id) }
```

Comment:
Still got a bunch of rockets up in the `let` statements. Can you swap 'em out so that we're consistent?